config/core/resources/containersource.yaml

```diff
@@ -31,13 +31,8644 @@ spec:
       subresources:
         status: {}
       schema:
-        openAPIV3Schema:
+        openAPIV3Schema: &openAPIV3Schema
           type: object
-          # this is a work around so we don't need to flush out the
-          # schema for each version at this time
-          #
-          # see issue: https://github.com/knative/serving/issues/912
-          x-kubernetes-preserve-unknown-fields: true
+          description: 'ContainerSource is an event source that starts a container image which generates events under certain situations and sends messages to a sink URI'
+          properties:
+              spec:
```

Comment:
One of the things I noted from the generator is that it is creating one more identation than necessary. Note spec is two levels down from properties in line 38 and ceOverrides is down two from properties in line 41. This extra identation will add up and might result in the file having more lines than necessary.